packages/ui/src/main/home-page/add-button/index.tsx

```
@@ -0,0 +1,254 @@
+  };
+
+  const getPositionStyle = () => {
+    switch (position) {
+      case 'bottom-center':
+        return {
+          position: 'fixed',
+          bottom: 24,
+          left: '50%',
+          transform: 'translateX(-50%)',
+        };
+      case 'bottom-right':
```

Biome (1.9.4)

[error] Useless case clause.

because the default clause is present:

Unsafe fix: Remove the useless case.


(lint/complexity/noUselessSwitchCase)